RubeesEat/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM mcr.microsoft.com/dotnet/sdk:8.0 AS build-env
+WORKDIR /App
+ 
+# Copy everything
+COPY . ./
+# Restore as distinct layers
+RUN dotnet restore
+# Build and publish a release
+RUN dotnet publish -c Release -o out
+ 
+# Build runtime image
+FROM mcr.microsoft.com/dotnet/aspnet:8.0
+WORKDIR /App
+COPY --from=build-env /App/out .
+ENTRYPOINT ["dotnet", "RubeesEat.dll"]
+
+# to build: docker build -t <name> .
+# to run: docker run -p <port(use port above 1024)>:8080 <name> .
```

Comment:
Personal pref. but I would put that up to to make it the first thing you see when opening the file